webapp/src/components/header/settings-dialog/SettingSection.tsx

```diff
@@ -30,6 +31,9 @@ export const SettingSection: React.FC<ISettingsSectionProps> = ({ setting, conte
     const onFeatureChange = useCallback(
         (featureKey: FeatureKeys) => {
             dispatch(toggleFeatureFlag(featureKey));
+            if (featureKey === FeatureKeys.MultiUserChat) {
+                dispatch(toggleMultiUserConversations({}));
```

Comment:
remove as this is not an arg

do you mind also removing the `Slice<ConversationsState>` typing on line 16 in `conversationsSlice`? this is what is allowing passing in incorrect args